### DISCUSSION TITLE
Add photos to different locations (USER = Location)

### DISCUSSION THREAD

**Discussion by cryd-s** [immich-app/immich#14621] [General]
Hi there i have installed immich in unraid and there i have folders like

/mnt/user/Selina/Fotos
/mnt/user/Manu/Fotos
/mnt/user/Nadine/Fotos

i want to use the upload from immich to that specific folders but dont know how to do this.

How can i create a upload location per user? 
If this would work i can pulish this to my docker.

**Comment by bo0tzz** [comment]
You can apply the concepts from https://immich.app/docs/guides/custom-locations but mount the users' paths instead.

**Reply by cryd-s** [reply]
but how can u mount a specific folder as user upload location

**Reply by cryd-s** [reply]
![image](https://github.com/user-attachments/assets/a9104e25-fc8d-47d9-a7d9-1132aa091251)

cannot find the folders like described 

![image](https://github.com/user-attachments/assets/60a92cc3-a891-45c3-94ea-26d9652d6a6f)

**Reply by rosbeef** [reply]
Did you find a solution ? I search a same way to sort photos in yunohost shared multimedia folders by users.
